### DISCUSSION TITLE
好奇是只有electron主进程编译为字节码了还是说渲染进程也可以编译

### DISCUSSION THREAD

**Discussion by sunny-fun** [alex8088/electron-vite#68] [Q&A] [answered]
好奇是只有`electron`主进程编译为字节码了还是说渲染进程也可以编译  就好比是用vue开发的 vue打包的js也进行jsc编译了吗

**Comment by alex8088** [comment, marked as answer]
字节码是基于node的vm模块实现，字节码需要运行在node环境，主进程和预加载脚本符合这个条件(预加载脚步要使用还需禁用沙盒)，渲染进程是基于浏览器环境因此不支持。
